testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/Standard.java

```diff
@@ -121,7 +121,7 @@ private DockerContainer createPrestoMaster()
     @SuppressWarnings("resource")
     private DockerContainer createTestsContainer()
     {
-        DockerContainer container = new DockerContainer("ghcr.io/trinodb/testing/centos7-oj8:" + imagesVersion, TESTS)
+        DockerContainer container = new DockerContainer("ghcr.io/trinodb/testing/centos7-oj11:" + imagesVersion, TESTS)
```

Comment:
Since the images are now going to have both JDKs installed does it make sense to still name images after JDK version?

Reply:
Yes. Oj11 is a default one still (and that's semantics of the image name)